geocodingv7/reversegeocoding.go

```diff
@@ -0,0 +1,43 @@
+package geocodingv7
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+// ReverseGeocoding allows reverse geocode from geo-positions to addresses.
```

Comment:
```suggestion
// ReverseGeocoding takes a set of geo-positions and returns their corresponding addresses.
```
Used plural (set of, addresses) in this suggestion, if it only takes one geo-position and returns one address, the suggestion should be updated accordingly.